### ISSUE TITLE
~topical reviewers?

### ISSUE THREAD

**Issue by abathur** [timokau/marvin-mk2#110]
I saw someone in #nixos ask specifically for a reviewer comfortable with C, and it just made me wonder if the ability to summon another reviewer who knows more about a specific language or software stack would be a useful ability for marvin in the longer term.

I would guess it's actually a reviewer command, so that it's only used if it poses domain-specific questions that a general reviewer can't answer.

**Comment by timokau** [comment]
Yes, I agree that it would be useful. We already use some [teams](https://github.com/orgs/NixOS/teams) for a similar purpose, but pinging one of those always notifies all members. Marvin could have a command that just pings one member of an expert team.

If we ignore rate limiting (or use some very simple approach without retry), that should be easy to implement. I will focus on the core functionality of marvin for now though. Progress is very slow right now, so don't hold your breath. It might take a long time, or it might never happen.

If you want to contribute, I'd gladly help you to get a working test setup.
